docs/Developers/Newcomers_Guide.rst

```diff
@@ -383,6 +383,25 @@ and clicking ``New Pull Request``.
 
 **Congratulations!** You have now created your first ``Pull Request``!
 
+Now, before having other contributors review your work, the system will do an
+automated code review and will comment on your Pull Request.
+You may be required to enhance your Pull Request by modifying your branch on
+your fork.
+
+.. important::
+
+    It is **never** necessary to close your Pull Request. **Never!**
```

Comment:
Necessary doesn't feel like a good word here. Maybe we can try `required` or `allowed`